serenata_toolbox/datasets/__init__.py

```diff
@@ -0,0 +1,73 @@
+import os
+
+from serenata_toolbox.datasets.downloader import Downloader
+from serenata_toolbox.datasets.local import LocalDatasets
+from serenata_toolbox.datasets.remote import RemoteDatasets
+
+
+class Datasets:
+    """
+    This is a wrapper for three different classes together handles the datasets
```

Comment:
I'm kind of unsure about a detail — what's the recommended place for that docstring about `Dataset('path-to-data')`: under `class Datasets` or under `def __init__`?